### PR TITLE
use images names and versions from manifest file for particular version

### PR DESCRIPTION
**Pull request by riuvshin**
needed for: https://github.com/codenvy/codenvy/issues/1156
- compose now used images with versions from `images` manifest file.
- improved dev mode, now it prints really needed puppet output to console.
- remove unnecessary sed commands
@TylerJewell @garagatyi Please take a look.

### CODE REVIEW

File: dockerfiles/cli/cli.sh [hunk 1]
```diff
@@ -832,18 +834,15 @@ cmd_config() {
   fi
 
   # Run the docker configurator
-  generate_configuration_with_puppet 
+  if [ "${CODENVY_DEVELOPMENT_MODE}" = "on" ]; then
+    # generate configs and print puppet output logs to console if dev mode is on
+    generate_configuration_with_puppet
+  else
+    generate_configuration_with_puppet >> "${LOGS}"
+  fi
```
Comment (TylerJewell) [inline dockerfiles/cli/cli.sh]: Why did you add this back in?  I removed this "if" logic earlier because it was suppressing certain output that was important.

**Comment by TylerJewell** [comment]
This is a very slick way to make sure that the docker-compose images.  But we should discuss the if logic around the configurator for dev mode.

File: modules/compose/templates/docker-compose.yml.erb [hunk 2]
```diff
@@ -83,9 +83,9 @@ services:
 # DOCKER REGISTRY
 ########################
   registry:
-    image: registry:2.5.0
+    image: <%= ENV["IMAGE_REGISTRY"] %>
     env_file:
-      - '<%= scope.lookupvar('compose::codenvy_folder') -%>/config/registry/registry.env'
+      - '/codenvy/instance/config/registry/registry.env'
```
Comment (TylerJewell) [inline modules/compose/templates/docker-compose.yml.erb]: By hard coding this AND removing the sed commands, we lose the ability to generate two docker-compose.yml files. If we want to generate docker-compose-container.yml and docker-compose-host.yml at the same time, we need to use sed to change the env file.s

File: dockerfiles/cli/cli.sh (hunk 1 above)
Comment (TylerJewell) [inline dockerfiles/cli/cli.sh]: This is ok to leave.

**Comment by TylerJewell** [comment]
I am a +1 for merge.  We will solve issues with doing multiple compose templates in another issue.

File: dockerfiles/cli/cli.sh (hunk 1 above)
Comment (riuvshin) [inline dockerfiles/cli/cli.sh]: this is really needed for dev mode, dev don't want to go to CLI logs to see puppet logs which is really needed to see is their changes applied properly

Comment (garagatyi) [inline dockerfiles/cli/cli.sh]: +1 it is needed
